pkg/api/handlers/compat/networks.go

```diff
@@ -182,27 +194,20 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 
 	network.Options = make(map[string]string)
 
-	// TODO: we should consider making this constants in c/common/libnetwork/types
+	// dockers networks are always isolated from each other
+	network.Options[nettypes.IsolateOption] = "true"
+
 	for opt, optVal := range networkCreate.Options {
 		switch opt {
-		case "mtu":
+		case nettypes.MTUOption:
 			fallthrough
 		case "com.docker.network.driver.mtu":
-			if network.Driver == nettypes.BridgeNetworkDriver {
-				network.Options["mtu"] = optVal
-			}
-		case "icc":
```

Comment:
Why was this removed?

Reply:
see commit message icc is not supported by us, it can just fall into the default case.